### PR TITLE
 feat: added footer and main component

### PR DESCRIPTION
**Pull request by A91y**
Added footer and main component
Replaced component structure from /components/.../main.tsx ->  /components/.../index.tsx to improve import statements.

### CODE REVIEW

**Comment by ShreyaPrasad1209** [comment]
@A91y can you add screenshot of how the UI looks like?

File: src/components/BloomMain/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { BloomComponent } from './styled';
+
+export const BloomMain = () => {
```
Comment (ShreyaPrasad1209) [inline src/components/BloomMain/index.tsx:3]: can we rename this to LandingPage?

File: src/components/BloomMain/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { BloomComponent } from './styled';
```
Comment (ShreyaPrasad1209) [inline src/components/BloomMain/index.tsx:1]: Also rename this to `LandingPageContainer`

File: src/components/BloomMain/styled.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const BloomComponent = styled.main`
```
Comment (ShreyaPrasad1209) [inline src/components/BloomMain/styled.tsx:3]: Rename this to LandingPageContainer. Also, awesome for using main and not div 🎉

File: src/components/BloomMain/styled.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  background-color: white;
+  color: #333;
+  text-align: center;
+  display: flex;
+  //   box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
```
Comment (ShreyaPrasad1209) [inline src/components/BloomMain/styled.tsx:8]: please remove this commented code.

File: src/components/Footer/styled.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const FooterContainer = styled.footer`
```
Comment (ShreyaPrasad1209) [inline src/components/Footer/styled.tsx:3]: good usage of footer element 👏🏻

File: src/components/NavigationBar/index.tsx
Comment (ShreyaPrasad1209) [inline src/components/NavigationBar/index.tsx:1]: Keep it `main.tsx` only. The main logic of a component sits inside `main.tsx`. Please rename every `index.tsx` to `main.tsx`

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
I've added few review comments, please take a look 👀 
Overall, great improvement from previous PR 😄

File: src/components/NavigationBar/index.tsx
Comment (A91y) [reply src/components/NavigationBar/index.tsx:1]: Resolved in commit [168b6d0](https://github.com/blossomlabsio/bloom/pull/10/commits/168b6d0ba38931639b80a447adc85fbc03f7d74f)

File: src/components/BloomMain/styled.tsx (hunk 4 above)
Comment (A91y) [reply src/components/BloomMain/styled.tsx]: Resolved in commit [f12579b](https://github.com/blossomlabsio/bloom/pull/10/commits/f12579b1b17efc52a1cd0041a8006c1bab1c96e2)

File: src/components/BloomMain/index.tsx (hunk 1 above)
Comment (A91y) [reply src/components/BloomMain/index.tsx]: Resolved in commit [042b677](https://github.com/blossomlabsio/bloom/pull/10/commits/042b6778760b8a976b52cec6be143c25f3c3911c)

File: src/components/BloomMain/index.tsx (hunk 2 above)
Comment (A91y) [reply src/components/BloomMain/index.tsx]: Resolved in commit [042b677](https://github.com/blossomlabsio/bloom/pull/10/commits/042b6778760b8a976b52cec6be143c25f3c3911c)

**Comment by A91y** [comment]
Implemented all the requested changes.

**Comment by A91y** [comment]
> @A91y can you add screenshot of how the UI looks like?
Here is the Screenshot
image

**Review by ShreyaPrasad1209 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**